TUTORIAL.md

```diff
@@ -786,6 +765,8 @@ but you don't need that for a Minikube deployment.
 
 8. Run `minikube service blog` to view the blog application. You should be able to login and add blogs, entries, and products.
 
+// I'm seeing a lot of list styles in this post that don't read like you, and aren't punctuated the way you typically punctuate lists. Is any of this post copy-pasted from elsewhere?
```

Comment:
Yes: https://github.com/mraible/jhipster4-demo/blob/master/README.adoc and http://www.jhipster-book.com/#!/news/entry/build-jhipster-microservices-and-deploy-to-google-cloud-with-kubernetes